deployment/roles/deploy/vars/main_template.yml

```diff
@@ -0,0 +1,10 @@
+---
+# This file is to be copied to ./main.yml (which is .gitignore-d)
+# and re-define values from defaults/main.yml you want to change.
+# For example for deploying to local cluster you'd use the following:
+
+project: myproject
```

Comment:
nit: can we change this to dist2src-update-stg?

Reply:
What would be the use case?

You'd need to specify `project` here only in case you'd deploy to different than default (defaults/main.yml) cluster/project.
The only use-case I see is deploying to local cluster (`oc cluster up`), where the only project is named `myproject`.
And we don't have any `dist2src-update-stg` project in the cluster.

Reply:
ah, okay, got it, I thought we'd have this file for every env